### PR TITLE
Remove makefile, move commands to package.json

### PR DESCRIPTION
**Pull request by jchavarri**
Proposal to put all commands in `package.json`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,17 +23,19 @@ point of the sample React app is [`src/ReactApp.re`](src/ReactApp.re).
 
 ## Commands
 
-In opam / dune projects, our personal preference is to group commonly used commands in a Makefile. This is completely optional.
+All the build commands are defined in the `package.json` `scripts` field. This
```
Comment (feihong) [inline README.md:26]:
```suggestion
All the build commands are defined in the `scripts` field of `package.json`. This
```

File: README.md [hunk 2]
```diff
@@ -34,17 +23,19 @@ point of the sample React app is [`src/ReactApp.re`](src/ReactApp.re).
...
+is completely optional, and other tools like `make` could be used.
 
-You can see all available commands by running `make help` or just `make`. Here
-are a few of the most useful ones:
+You can see all available commands by running `npm run`. There are explanations
```
Comment (feihong) [inline README.md:29]: I see that running `npm run` only shows the script name and the corresponding commands. I guess `scriptsComments` is just a convention, and is totally ignored by `npm run`?

**Review by feihong — APPROVED** [review]
I think the changes make sense because npm scripts are more familiar to frontend developers than Makefiles.

**Comment by maxkorp** [comment]
Small bug I noticed `npm init` ends with `npm run install` but there is no install script. Should that be `npm run install-opam-npm`?

File: README.md (hunk 2 above)
Comment (jchavarri) [reply README.md:29]: Yeah `scriptsComments` is just a hacky way to somehow document these commands. As they also appear in the README, are listed in `npm run`, and then somehow documented with this `scriptsComments` hack, I hope that with all this we can somehow replace the old `make help` behavior 😄

**Comment by jchavarri** [comment]
@maxkorp thanks, should be fixed in https://github.com/melange-re/melange-opam-template/pull/33/commits/58682d5cd62d466b9e97d41a040905bec8686b26 (also updated README where `npm run install` was also being mentioned).

**Pull request merged**